src/main/java/org/troyargonauts/subsystems/LEDSystem.java

```diff
@@ -0,0 +1,104 @@
+package org.troyargonauts.subsystems;
+
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdleConfiguration;
+import com.ctre.phoenix.led.RainbowAnimation;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+
+public class LEDSystem extends SubsystemBase {
```

Comment:
Add class description Javadoc

Reply:
Try something like "This class uses REV's CANdle to control the LEDs on the robot"